frontend_app_file/src/css/index.styl

```diff
@@ -1,7 +1,7 @@
 @import "../../node_modules/tracim_frontend_lib/src/css/Variable.styl"
 
 .file
-  height calc(100% - 85px)
+  height calc(100% - 61px)
```

Comment:
This cannot be a variable in stylus ?

Reply:
well, it could but it would means a variable in the file Variable.styl of frontend_lib.
But it's the frontend's Header component design that will define the value of that variable to be then used by all apps.

That means the frontend's Header component have a design that ends up having a total height. Then this total height should be set in the frontend_lib variable then be used by components.

So in the end, we can't have everything that depends on one single variable. The Header component cannot depends on it since it's the one that defines it.

So I'm not sure what would be best

Reply:
Please open an issue about possible refacto about applications size / container problematic. Then merge this MR.

Reply:
see https://github.com/tracim/tracim/issues/1657